### ISSUE TITLE
[Custom Fields] Still no decision about Mustache or similiar

### ISSUE THREAD

**Issue by zero-24** [joomla/joomla-cms#12623]
### Steps to reproduce the issue

There is still no decision about Mustache or similiar
### Actual result

> I'v removed Mustache from the core for now. Content plugins and per field output is not working anymore. I'v opened a new PR https://github.com/joomla-projects/custom-fields/pull/135 which reverts the change. This PR is intended as the starting point for a new common parser functionality for content plugins (if somebody has time to do it). At least we don't hold back the custom fields project. Hope this procedure is ok for all.

https://github.com/joomla-projects/custom-fields/issues/12#issuecomment-226998842
### Additional comments

Other way arround proposal is: https://github.com/joomla/joomla-cms/pull/11702

**Comment by zero-24** [comment]
Closing as we are going zo remove that functionality with content plugins completly.

**Comment by chrisdavenport** [comment]
I do hope that the content plugin functionality (regardless of whether it's based on Moustache, or whatever) can be re-introduced, perhaps in 3.8, as it's one of the most powerful features of the custom fields extension.

**Comment by zero-24** [comment]
Maybe but there are serveral PRs out to remove that functionality and there is no decision by the PLT. So i think first we need a decision there before we can implement such a feature and i know that it is no easy descision.
